participants/filecoin.net-disk.md

```diff
@@ -0,0 +1,56 @@
+# <Project Name>
+Filecoin.Net Disk
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+My project name is Filecoin.Net Disk
+My slack username is cannon, slack emil is cannon3758@outlook.com
+
+## (1) Brief Project Description
+This project is provide data store for users, which is use web UI interface to operation.
+And then we use lotus for data storage and retrieve deal from filecoin blockchain 
+
+
+## (2) Link to Project UI
+http://183.60.252.190:3334
```

Comment:
Hi there cannon! When we look up the hashes on this site on filfox, no result is returned. Is this website currently just containing a mockup or is the site live?

Reply:
The hash is make from lotus import XXX command,  It may not be found in filfox， but you can definitely find it throughout the blockchain